tests/contract_testcases/solana/functions/push_pop_fixed_length_array.sol

```diff
@@ -0,0 +1,20 @@
+contract MyTest {
+    function  foo1() public pure returns (uint256)  { 
+        uint64[10] a; 
+        a[9]  =  0x41; 
+        a.push(2); 
+        return  (a[9]); 
+    }
+
+    function foo2() public pure returns (uint256) {
+        uint64[10] a; 
+        a[9]  =  0x41; 
+        a.pop(); 
+        return  (a[9]);
+    }
+}
+
+
+// ---- Expect: diagnostics ----
+// error: 5:11-15: method 'push' does not exist
```

Comment:
a nicer diagnostic saying that push and pop are not available for fixed sizes array would be preferable

Reply:
Let me check how to fix this.

Reply:
The function call matching algorithm needs improvements. In the way it is now, it is not possible to say that push and pops are not available for external calls, because adding an error to diagnostics prevents the algorithm from finding a match for the library function. Likewise, trying the library case first makes other test cases fail.

Reply:
I agree, briefly looked at it and came to the same conclusion. Just matching on the function name is not gonna cut it.

Reply:
Now fixed.